src/dvclive/studio.py

```diff
@@ -53,7 +53,7 @@ def _adapt_images(live):
     return {
         _adapt_path(live, image.output_path): {"image": _adapt_image(image.output_path)}
         for image in live._images.values()
-        if image.step > live._latest_studio_step
+        if image.step >= live._latest_studio_step
```

Comment:
same logic = same change ^^

Reply:
It is not needed for the PR tho. Tell me what is the policy. Should I change it to `>=` or keep it as `>` ?